release-please-config.json

```
@@ -19,6 +19,7 @@
     "packages/sdk-node": { },
     "packages/sdk-rtl": { "release-as": "" },
     "packages/wholly-sheet": { "release-as": "" },
-    "python": { "release-type": "python", "package-name": "looker_sdk" }
+    "python": { "release-type": "python", "package-name": "looker_sdk" },
+    "go": {	"release-type": "go", "package-name": "go", "tag-separator": "/" }
```

woohoo!!

convert tab to space:
```suggestion
    "go": { "release-type": "go", "package-name": "go", "tag-separator": "/" }
```

also, not sure how go package naming works but I think what you have here will literally create a tag named `go/v22.4.0` and a release named `go go/v22.4.0` (assuming release-please creates tags/release like it does for the other languages). Is that what you want? or maybe `looker_go`?

edit: now that I'm reading your publish model I see that the tag name `go/v22.4.0` is what you actually want

---

Updated, thanks! I love this commit suggestion UX. 

And yeah, it's not ideal. I would prefer the release to be named `go-sdk go/v0.0.1`, but release-please is not that configurable, and I have to have the tag be in the format of `go/v0.0.1` unfortunately. :(

---

Oh I missed adding a `"release-as": ""`. D'oh!

---

well, since it's going into the next release, we'll have a top-level one anyway so no need for a specific `go` release-as

---

My bad if i wasn't clear in the PR description. I'm keeping the Go SDK at v0 and making use of the *-pre-major* flags to keep it below v1. We're not ready for v1+ versioning for Go SDK yet.

---

It doesn't 😭  It has to be the subdirectory of the repo. So `go` and then a `/` and then version _with_ a v prefixed. Argh go is super opinionated. I've done too much reading into Go versioning and their motivations. _sigh_

https://go.dev/ref/mod
`If the module is not defined in the repository’s root directory, the module subdirectory is the part of the module path that names the directory, not including the major version suffix. This also serves as a prefix for semantic version tags. `

---

That's totally fine. Getting it version tagged is splendid however we have to make it work.